scripts/listener.ts

```diff
@@ -95,6 +98,21 @@ class StandaloneEventHandler extends IEventHandler {
 
 const skipCatchup = false;
 
+const tokenListUrls = [
+  "https://wispy-bird-88a7.uniswap.workers.dev/?url=http://tokenlist.aave.eth.link",
+  "https://gateway.ipfs.io/ipns/tokens.uniswap.org",
+  "https://wispy-bird-88a7.uniswap.workers.dev/?url=http://defi.cmc.eth.link"
+]
+
+async function getTokenLists() {
```

Comment:
Please lint this function/the rest of this file. We should add it to the lint files list tbh.